apipie-rails.gemspec

```diff
@@ -38,6 +38,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop-rails'
   s.add_development_dependency 'rubocop-rspec'
   s.add_development_dependency 'rubocop-performance'
+  s.add_development_dependency 'rubocop-rspec_rails'
```

Comment:
can you try adding ` if RUBY_VERSION >= "2.7"` please?

or take out this line for now, ... if you can

Reply:
I can do that, but the problem will probably then be that ruby 2.6 rubocop will complain about `RSpecRails/InferredSpecType` being a cop it doesn't understand whereas ruby 2.7+ rubocop complains about `RSpec/Rails/InferredSpecType`.  Can we put conditional rubocop_todo by ruby version?

In another thread I think you suggested someone had dropped ruby 2.6 / rails 5 support - should we make that official?

Reply:
I'll push up a commit with the `if` version comparison so we can test that theory though